src/main/java/net/minecraftforge/common/Tags.java

```diff
@@ -416,6 +420,19 @@ private static void init(){}
         }
     }
 
+    public static class EntityTypes
+    {
+        private static void init() {}
+
+        public static final Tags.IOptionalNamedTag<EntityType<?>> BOSSES = tag("bosses");
+        public static final Tags.IOptionalNamedTag<EntityType<?>> CONSTRUCTS = tag("constructs");
```

Comment:
What is constructs? You have elder guardian and guardian along with iron and snow golems in here. I'm unsure what this tag would mean. If you're talking about entities that can be created, then the former two wouldn't be valid. Additionally, ender dragons or villagers could be created as well. If you're talking about entities that can spawn in structures, then there is a massive list missing, snow golems not one of them which spawn in a structure.

Reply:
This seems rather niche and something I think would best fit as a tag for your specific mod (if you even have a use). What would be the use case of such a tag existing in forge?

Reply:
There is no real reason. I just saw that Forge didn't have any preexisting entity tags generated in it, so I decided to add one or two. I'll go ahead and remove it and just leave the bosses tag.